src/Controller/HealthController.php

```diff
@@ -27,7 +28,7 @@
  * The health controller is used to display the health of the application. Information is returned as a JSON response.
  * When one of the health checks (run by the HealthCheckerChain) fails, the DOWN message of that check is shown.
  */
-class HealthController
+class HealthController extends AbstractController
```

Comment:
Why extend the `AbstractController`? We use none of it's features. But maybe this is a framework obligation introduced in a newer Symfony version than I'm acquainted with? If possible, please remove this extension. This also applies for the InfoController

Reply:
I changed the code to use the default json return $this->json(...) from the AbstractController.
In my eyes it is always good to use the default symfony features. But i can revert the json responsens and the extends AbstractController if you wish. Please let me know

Reply:
They removed this code in the service.yaml (after i upgraded the recipes in an other project to symfony 5.4.dev)
I guess it is still possible to manuel tag Services as controllers. But extend from AbstractController is the safer way.

`# controllers are imported separately to make sure services can be injected
  # as action arguments even if you don't extend any base controller class
    App\Controller\:
        resource: '../src/Controller/'
        tags: ['controller.service_arguments']
`

Reply:
Personally I prefer to couple as loosely to Symfony as possible. Using the `json` feature provided by the abstract controller is a kind of legitimate argument to use the abstract controller. Feel free to leave this as is!